templates/engage/cloud-fray.html

```diff
@@ -0,0 +1,34 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}How to face the challenges of becoming a cloud operator{% endblock %}
+
+{% block meta_description %}In a shifting market, the traditional revenues of telcos and service providers are under threat. Voice revenues are at an all-time low, and high wireless data costs are negatively impacting margins.{% endblock %}
+
+{% block meta_copydoc %}https://app-sjg.marketo.com/#LP5496A1LA1{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="How to face the challenges of becoming a cloud operator" image="9f61b97f-logo-ubuntu.svg" url="#the-form" cta="Download the eBook" %}
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+        <p>In a shifting market, the traditional revenues of telcos and service providers are under threat. Voice revenues are at an all-time low, and high wireless data costs are negatively impacting margins. As an additional challenge, many enterprises are moving low-value workloads and applications to public clouds such as Amazon, eroding traditional managed hosting revenues.</p>
```

Comment:
please use 2 spaces indentation instead of 4 as this is a Canonical best practice https://canonical-web-and-design.github.io/practices/coding/html.html